rest_framework/fields.py

```diff
@@ -576,14 +576,18 @@ def __init__(self, **kwargs):
             self.validators.append(MinLengthValidator(self.min_length, message=message))
 
     def run_validation(self, data=empty):
+        (is_empty_value, data) = self.validate_empty_values(data)
+        if is_empty_value:
+            return data
+
+        value = self.to_internal_value(data)
```

Comment:
Forget about the previous comment. Actually, it's the call to to_internal_value that does the trimming.